.doc_gen/metadata/serverless_metadata.yaml

```diff
@@ -14,6 +14,54 @@ serverless_S3_Lambda:
             - description: Consuming an S3 event with Lambda using JavaScript.
               snippet_files:
                 - integration-s3-to-lambda/example.js
+    Python:
+      versions:
+        - sdk_version: 2
+          github: https://github.com/aws-samples/serverless-snippets/tree/main/integration-s3-to-lambda
+          excerpts:
+            - description: Consuming an S3 event with Lambda using Python.
+              snippet_files:
+                - integration-s3-to-lambda/example.py
+    Java:
+      versions:
+        - sdk_version: 2
+          github: https://github.com/aws-samples/serverless-snippets/tree/main/integration-s3-to-lambda
+          excerpts:
+            - description: Consuming an S3 event with Lambda using Java.
+              snippet_files:
+                - integration-s3-to-lambda/Handler.java
+    .NET:
+      versions:
+        - sdk_version: 2
+          github: https://github.com/aws-samples/serverless-snippets/tree/main/integration-s3-to-lambda
+          excerpts:
+            - description: Consuming an S3 event with Lambda using .NET.
+              snippet_files:
+                - integration-s3-to-lambda/Function.cs
+    Go:
+      versions:
+        - sdk_version: 2
+          github: https://github.com/aws-samples/serverless-snippets/tree/main/integration-s3-to-lambda
+          excerpts:
+            - description: Consuming an S3 event with Lambda using Go.
+              snippet_files:
+                - integration-s3-to-lambda/main.go
+    TypeScript:
+      versions:
+        - sdk_version: 2
+          github: https://github.com/aws-samples/serverless-snippets/tree/main/integration-s3-to-lambda
+          excerpts:
+            - description: Consuming an S3 event with Lambda using TypeScript.
+              snippet_files:
+                - integration-s3-to-lambda/example.ts
```

Comment:
There's no TypeScript SDK, so either remove this or add the excerpt under JavaScript.